<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LT0055527" failures="0" tests="5" name="enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest" time="161.941" errors="1" timestamp="2020-12-17T16:23:47 IST" skipped="0">
  <testcase classname="enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest" name="OIMAuthentication" time="5.150"/>
  <system-out/>
  <testcase classname="enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest" name="CAFOptionSearch" time="11.004"/>
  <system-out/>
  <testcase classname="enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest" name="SelectingCAFType" time="2.603"/>
  <system-out/>
  <testcase classname="enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest" name="LocationIDSearch" time="24.476"/>
  <system-out/>
  <testcase classname="enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest" name="CAFEntry" time="118.708">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)&quot; because the return value of &quot;java.lang.ThreadLocal.get()&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "java.lang.ThreadLocal.get()" is null
at enterprise.mobility.Listeners.logCAFNumber(Listeners.java:82)
at enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest.CAFEntry(Cls_OrderPlacementAtDataEntryStageTest.java:237)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CAFEntry -->
  <system-out/>
</testsuite> <!-- enterprise.mobility.Cls_OrderPlacementAtDataEntryStageTest -->
